src/Files.App/UserControls/Widgets/QuickAccessWidget.xaml.cs

```diff
@@ -284,6 +286,8 @@ private async void ModifyItem(object? sender, ModifyQuickAccessEventArgs? e)
 					{
 						var item = await App.QuickAccessManager.Model.CreateLocationItemFromPathAsync(itemToAdd);
 						var lastIndex = ItemsAdded.IndexOf(ItemsAdded.FirstOrDefault(x => !x.IsPinned));
+						if (ItemsAdded.Any(x => x.Path == itemToAdd))
+							continue;
```

Comment:
Should there be an empty line after this?

Reply:
I'll add it if you prefer